app/views/dashboard/_eeo_data.haml

```diff
@@ -4,7 +4,7 @@
     .note_item
       = select_tag :department_ids,
       options_from_collection_for_select(@departments, :id, :name),
-      class: 'dept_select', multiple: true
+      class: 'dept-select', multiple: true
```

Comment:
Okay, I was wrong, scratch that. Underscores versus dashes do not make a difference. I'm leaving it as a dash because it does not matter. 
I had changed it in the first place because I made the JS line a dash and the class name an underscore--it was a typo, and caused things not to work, and I was freaking out and then found the error.... changed it to a dash to make them match.